src/components/relationships-panel/RelationshipsPanel.vue

```diff
@@ -0,0 +1,141 @@
+              :value="RelationshipTypes.SHAREHOLDER" :label="RelationshipTypes.SHAREHOLDER"
+              :rules="relationshipRules" />
+          </v-col>
+
+          <v-col cols="4">
+            <v-checkbox id="court-ordered-party-checkbox" class="mt-0" v-model="selectedRelationships"
+              :value="RelationshipTypes.COURT_ORDERED_PARTY" :label="RelationshipTypes.COURT_ORDERED_PARTY"
+              :rules="relationshipRules" />
+          </v-col>
+        </v-row>
+      </v-card>
+    </div>
+  </v-card>
+</template>
+<script lang="ts">
+
+import Vue from 'vue'
+import { Component, Watch, Emit, Prop } from 'vue-property-decorator'
+
+@Component({})
+export default class RelationshipsPanelShared extends Vue {
+  /** Draft restoration relationships */
+  @Prop({ default: ['Heir or Legal Representative'] }) readonly draftRelationships!: []
+
+  // Local properties
+  private selectedRelationships = []
+  private relationshipRules = []
+
+  // Relationship Types Enum for the template
+  RelationshipTypes = {
+    COURT_ORDERED_PARTY: 'Court Ordered Party',
+    DIRECTOR: 'Director',
+    HEIR_LEGAL_REP: 'Heir or Legal Representative',
+    OFFICER: 'Officer',
+    SHAREHOLDER: 'Shareholder'
+  }
+
+  /**
+   * Called when component is mounted.
+   * Automatically check all previously checked relationships (if any) when user continues a draft.
+   */
+  mounted (): void {
+    if (this.draftRelationships !== null) {
+      this.selectedRelationships.push(...this.draftRelationships)
+    } else {
+      this.selectedRelationships.push(this.RelationshipTypes.HEIR_LEGAL_REP)
+    }
+  }
+
+  /** The validation rules for the Relationships. */
+  private setRelationshipRules (): void {
+    this.relationshipRules = [
+      () => this.selectedRelationships.length > 0 || ''
+    ]
+  }
+
+  // Emit the selected relationships array.
+  @Emit('emitRelationshipsChanged')
+  private relationshipsChanged (): any[] {
+    return this.selectedRelationships
+  }
+
+  // Emit a boolean (validation) which is if at least one relationship is selected.
+  @Emit('emitRelationshipsValid')
+  private relationshipsValid (event: boolean): boolean {
+    return event
+  }
+
+   /**
+    * Emit events whenever the relationships array is changed (validation and the selected relationships array).
+    * Change the tooltip text to red (error) if no relationships are selected.
+    */
+   @Watch('selectedRelationships')
+  private setRelationships (val) {
+    this.setRelationshipRules()
+    this.$emit('emitRelationshipsChanged', this.selectedRelationships)
+    if (this.selectedRelationships.length > 0) {
+      this.$emit('emitRelationshipsValid', true)
+    } else {
+      this.$emit('emitRelationshipsValid', false)
+    }
+  }
+}
+
+</script>
+
+<style lang="scss" scoped>
+@import '@/assets/styles/theme.scss';
+
+#relationship-style {
+  padding-bottom: 1rem;
+  padding-left: 1rem;
+  margin-right: 1.5rem;
```

Comment:
For more layout styles, I have found it's better to use the [Vuetify spacing classes](https://vuetifyjs.com/en/styles/spacing/) right in the template -- when fixed or refactoring layouts/whitespace (something we have to do every once in a while), it's _way_ easier to see and tweak the classes in the template instead of having to look for the class and whatever it declares.